content/influxdb/v2.0/query-data/influxql.md

```diff
@@ -43,13 +43,60 @@ By default, the `/query` compatibility endpoint returns results in **JSON**.
 To return results as **CSV**, include the `Accept: application/csv` header.
```

Comment:
Nora, what do you think bout moving the part about mapping buckets up before running the query, and add numbered steps? Then, we can drop the "Database and retention policy mapping" and "Map a bucket to a database and retention policy" sections. Something. like this:

**Query data with InfluxQL**

In 1.x, data is stored in databases and retention policies. In 2.0, data is stored in buckets. To query a bucket with InfluxQL, the bucket must be mapped to a database and retention policy.

{{% note %}}
If data is written into InfluxDB 2.0 using the `/write` 1.x compatibility API, buckets were mapped automatically, so you can skip steps 1-2. For more information, see [Database and retention policy mapping service](...).
{{% /note %}}

1. Verify the buckets that you want to query are mapped to a database and retention policy: (include sample CURL command and link to API doc.)
2. To map unmapped buckets to a database and retention policy: (include sample CURL command and link to API doc.)
3. Use the 1.x `/query` endpoint to query a mapped bucket. Include the following parameters:....:
     For example:
     ```curl -G https://us-west-2-1.aws.cloud2.influxdata.com/query?db=db&rp=rp \
           -H "Authorization: Token YourAuthToken" \
           --data-urlencode "q=SELECT used_percent FROM mem WHERE host=host1"
     ```
   By default, the 1.x `/query` compatibility endpoint returns results in JSON. To return results as CSV, 
   include the Accept: application/csv header.